bin/xud

```diff
@@ -87,6 +87,16 @@ const { argv } = require('yargs')
       type: 'boolean',
       default: undefined,
     },
+    'p2p.discover': {
+      describe: 'Do not send GET_NODES if false, when connecting to a peer',
+      type: 'boolean',
+      default: false,
+    },
+    'p2p.detectExternalIp': {
```

Comment:
I think we decided on all lower case config options, so this should be `detectexternalip`.

Reply:
Alright, changing it